suite/contracts_flux_test.go

```diff
@@ -2,46 +2,60 @@ package suite
 
 import (
 	"context"
+	"math/big"
+	"strings"
+	"time"
+
 	"github.com/ethereum/go-ethereum/common"
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/ginkgo/extensions/table"
 	. "github.com/onsi/gomega"
 	"github.com/rs/zerolog/log"
 	"github.com/smartcontractkit/integrations-framework/client"
+	"github.com/smartcontractkit/integrations-framework/config"
 	"github.com/smartcontractkit/integrations-framework/contracts"
-	"github.com/smartcontractkit/integrations-framework/suite"
-	"github.com/smartcontractkit/integrations-framework/tools"
-	"math/big"
-	"strings"
-	"time"
+	"github.com/smartcontractkit/integrations-framework/environment"
 )
 
 var _ = Describe("Flux monitor suite", func() {
+	var conf *config.Config
+
+	BeforeEach(func() {
+		var err error
+		conf, err = config.NewWithPath(config.LocalConfig, "../config")
+		Expect(err).ShouldNot(HaveOccurred())
+	})
+
 	DescribeTable("Answering to deviation in rounds", func(
 		initFunc client.BlockchainNetworkInit,
 		fluxOptions contracts.FluxAggregatorOptions,
 	) {
-		s, err := suite.DefaultLocalSetup(initFunc)
+		network, err := initFunc(conf)
+		Expect(err).ShouldNot(HaveOccurred())
+		testEnv, err := environment.NewBasicEnvironment("basic-flux-monitor", 3, network)
```

Comment:
Stealing some ideas from @skudasov to compress the setup, but through the environment creation instead.